modules/ml/src/rtrees.cpp

```diff
@@ -422,6 +422,18 @@ Ptr<RTrees> RTrees::create()
     return makePtr<RTreesImpl>();
 }
 
+//Function needed for Python and Java wrappers
+Ptr<RTrees> RTrees::load(const String& filepath)
+{
+    FileStorage fs;
+    fs.open(filepath, FileStorage::READ);
+
+    Ptr<RTrees> rtree = makePtr<RTreesImpl>();
+
+    ((RTreesImpl*)rtree.get())->read(fs.getFirstTopLevelNode());
```

Comment:
I see the problem. We can leave `getFirstTopLevelNode()` to have compliance with the `write()` method, but generally this code won't work in case if FirstTopLevelNode doesn't contain the classifier.

In `read()` we can use any node and I think It's needed to introduce `nodeName` parameter into `load()` method. If it's not provided FirstTopLevelNode can be used by default.